test/system/030-run.bats

```diff
@@ -436,6 +436,16 @@ json-file | f
 @test "podman run --log-driver journald" {
     skip_if_remote "We cannot read journalctl over remote."
 
+    # We can't use journald on RHEL as rootless, either: rhbz#1895105
+    if is_rootless; then
+        run journalctl -n 1
+        if [[ $status -ne 0 ]]; then
+            if [[ $output =~ permission ]]; then
+                skip "Cannot use rootless journald on this system"
+            fi
+        fi
+    fi
```

Comment:
Would it be good to move this to a `skip_if_rootless_rhel` type of function?

Reply:
Sorry for the too-late response. As much as I hate duplication, on two-offs I often just hold my nose and leave it. Should there ever be a need for a third exception, then yes, absolutely, refactor.